### PR TITLE
Temporarily discard mis-assigned data on chunk boundary

### PR DESCRIPTION
**Pull request by JelleAalbers**
This is a temporary fix to be able to build data affected by https://github.com/coderdj/redax/pull/82. After that is deployed and the data is built, we should reverse this hack.

### CODE REVIEW

**Comment by JelleAalbers** [comment]
The build failure on python 3.6 also affects master, even though master built fine yesterday. So this must be due to a dependency update. Monday's round of automatic updates on base_environment will hopefully tell us more. Anyway, not a relevant issue for this I think.

**Comment by jorana** [comment]
@JelleAalbers should we maybe add it as an option that changes the lineage so that we can easily distinguish between runs with this option and those without?

I've just checked out this PR on eb3 (py3.7) and tested it on run 7251; it works.

File: straxen/plugins/daqreader.py [hunk 1]
```diff
@@ -156,6 +156,18 @@ def _load_chunk(self, path, start, end, kind='central'):
         first_start, last_start, last_end = None, None, None
         if len(records):
             first_start, last_start = records[0]['time'], records[-1]['time']
+
+            if last_start == end:
```
Comment (jorana) [inline straxen/plugins/daqreader.py:160]: Maybe do it only when a tracked option == ``True`` in order to change the lineage for now?

**Comment by JelleAalbers** [comment]
Thanks for testing! If we change the lineage people still won't be able to load these runs. We could add a non-tracked option for now, but I think we'll be turning it off forever on Monday once Darryl deploys https://github.com/coderdj/redax/pull/82.

**Comment by jorana** [comment]
Makes sense! I'll merge and update the ebs.

**Comment by jorana** [comment]
I cannot merge because of travis, I'll simply checkout this branch, we then also don't have to remove this temporary change from master again.

**Comment by JelleAalbers** [comment]
Sorry, hit the button before reading your second sentence :-)

**Pull request merged**